app/src/main/java/com/example/longdroid/presentation/notelist/NoteListViewModel.kt

```diff
@@ -0,0 +1,16 @@
+package com.example.longdroid.presentation.notelist
+
+import android.util.Log
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.example.longdroid.data.model.response.ResponseNoteList
+import com.example.longdroid.data.service.NoteListService
+import kotlinx.coroutines.launch
+import retrofit2.Response
+
+class NoteListViewModel(private val noteService: NoteListService) : ViewModel() {
+    val noteListLiveData: MutableLiveData<Response<ResponseNoteList>> = MutableLiveData()
```

Comment:
```suggestion
    val _noteListData = MutableLiveData<ResponseNoteList>()
    val noteListData : LiveData<ResponseNoteList> = _noteListData
```
MutableLiveData 밖에 없네요 LiveData 객체를 만들어주시고 이렇게 하는 이유는 backing properties 때문입니다!
세미나 자료에 잘 나와 있으니 다시 보시면 이해될거에요~